content/blog/2018-11-27-react-16-roadmap.md

```diff
@@ -32,7 +34,7 @@ We have a single vision for how all of these features fit together, but we're re
 
 The gradual release strategy helps us refine the APIs, but the transitional period when some things aren't ready can be confusing. Let's look at what these different features mean for your app, how they relate to each other, and when you can expect to start learning and using them.
 
-### [React 16.6](/blog/2018/10/23/react-v-16-6.html): Suspense for Code Splitting (shipped)
+### [React 16.6](/blog/2018/10/23/react-v-16-6.html) (shipped): The One with Suspense for Code Splitting
```

Comment:
This would have a nicer ring to it if we dropped "Suspense for". :-) But I want to emphasize the staggered rollout.